daemons/filter_merge/app.py

```diff
@@ -11,14 +11,17 @@
 reducer - Combine partial results into final .zarr file in S3, return S3 location
 """
 
-from matrix.lambdas.filter_merge.driver import driver
+from matrix.lambdas.filter_merge.driver import Driver
 from matrix.lambdas.filter_merge.mapper import mapper
 from matrix.lambdas.filter_merge.worker import worker
 from matrix.lambdas.filter_merge.reducer import reducer
 
 
-def driver_handler():
-    driver()
+def driver_handler(event, context):
+    # TODO: better error handling
```

Comment:
Do we want to start talking about logging/correlation id?

Reply:
It will be easier to add in standardized logging/correlation id one PR rather than scattering the approach in multiple PRS. Lets focus on that after a passing functional test